tools/celtest/test_runner.go

```diff
@@ -417,6 +419,17 @@ func fileDescriptorSet(path string) (*descpb.FileDescriptorSet, error) {
 	return fds, nil
 }
 
+// PartialEvalProgramOption returns a TestRunnerOption which enables partial evaluation for the CEL
+// program.
+//
+// Note: Partial evaluation where only root level variables are missing is supported.
```

Comment:
nit/clarity: you could just say you are using Env.PartialVars(activation) to infer unknown root vars based on the activation.